test_raw_fd_api fails on ubuntu sometimes

```
test test_raw_fd_api ... FAILED

failures:

---- test_raw_fd_api stdout ----
thread 'test_raw_fd_api' panicked at 'called `Result::unwrap()` on an `Err` value: PcapError("truncated dump file; tried to read 4 file header bytes, only got 0")', tests/lib.rs:238:34
stack backtrace:
   0: rust_begin_unwind
             at /rustc/397b390cc76ba1d98f80b2a24a371f708dcc9169/library/std/src/panicking.rs:475
   1: core::panicking::panic_fmt
             at /rustc/397b390cc76ba1d98f80b2a24a371f708dcc9169/library/core/src/panicking.rs:85
   2: core::option::expect_none_failed
             at /rustc/397b390cc76ba1d98f80b2a24a371f708dcc9169/library/core/src/option.rs:1221
   3: core::ops::function::FnOnce::call_once
   4: core::ops::function::FnOnce::call_once
             at /rustc/397b390cc76ba1d98f80b2a24a371f708dcc9169/library/core/src/ops/function.rs:227
note: Some details are omitted, run with `RUST_BACKTRACE=full` for a verbose backtrace.


failures:
    test_raw_fd_api

test result: FAILED. 6 passed; 1 failed; 0 ignored; 0 measured; 0 filtered out

error: test failed, to rerun pass '--test lib'
##[error]Process completed with exit code 101.
```
https://github.com/Trolldemorted/pcap/runs/1039588778?check_suite_focus=true

--- a/tests/lib.rs
+++ b/tests/lib.rs
@@ -248,7 +248,6 @@ fn test_raw_fd_api() {
             let mut file_in = File::open(&filename).unwrap();
             let mut file_out = unsafe { File::from_raw_fd(fd_out) };
             io::copy(&mut file_in, &mut file_out).unwrap();
-            assert_eq!(unsafe { libc::close(fd_out) }, 0);
         });
 
         // Open the capture with pipe's file descriptor

